src/ZfSimpleMigrations/Library/Migration.php

```diff
@@ -288,12 +296,7 @@ protected function applyMigration(array $migration, $down = false, $fake = false
                 $sqlList = $down ? $migrationObject->getDownSql() : $migrationObject->getUpSql();
                 foreach ($sqlList as $sql) {
                     $this->outputWriter->writeLine("Execute query:\n\n" . $sql);
-                    /** @var \Zend\Db\Adapter\Driver\Pdo\Statement $statement */
-                    $statement = $this->connection->prepare($sql);
-                    $statement->execute();
-                    /** @var \PDOStatement $statement */
-                    $statement = $statement->getResource();
-                    while ($statement->nextRowset()) {/* https://bugs.php.net/bug.php?id=61613 */}
+                    $this->connection->execute($sql);
```

Comment:
I reverted changes from a previous pull request https://github.com/vgarvardt/ZfSimpleMigrations/pull/8 because it broke support for Sqlite drivers (and any driver not supporting multiple row sets). I suggest handling multi-statement error checking as a driver specific feature via the feature set pattern used in TableGateway (http://framework.zend.com/manual/current/en/modules/zend.db.table-gateway.html) but that change will be rather large. For now I've reverted it back to how it was and updated README.md with a recommendation that you shouldn't send multi-statements to addSql.